src/main/java/deez/Deez.java

```diff
@@ -0,0 +1,231 @@
+package deez;
```

Comment:
Would it be better to categorise other classes into more specific packages instead of putting everything in deez?

Reply:
Yeah, that's a good idea! I'll probably get around to it while refactoring for JavaFX